backend/routes.js

```diff
@@ -589,10 +589,10 @@ export default class Servers {
         /* Plotly v2 API requests */
 
         server.post('/datacache', function getDatacacheHandler(req, res, next) {
-            const {payload, type: contentType} = req.params;
+            const {payload, requestor, type: contentType} = req.params;
```

Comment:
No need for `requestor` since this PR already adds it to `payloadJSON`

Reply:
Actually, Thats not True. The `payload` here just contains the CSV (or Plot) data.

See this: https://github.com/plotly/falcon-sql-client/pull/313/commits/4661d7561e4cb9ea5142ad9b2a70c5a329aabbf0#diff-70a485b1508d01fef5e75ca9aad77b28R182

`payload` and `requestor` are different Keys.

Reply:
isn't the username also stored in `payload.requestor`?

Reply:
It is not.

Reply:
Oh! I see confused `payload` and `payloadJSON` :fearful:  .https://github.com/plotly/falcon-sql-client/blob/264658595575f84ba9dcea44043835b6e75ca46d/app/components/Settings/Preview/Preview.react.js#L182-L183